specification/gedcom-03-datamodel.md

```diff
@@ -1046,6 +1046,17 @@ Optional isolated name parts; see `PERSONAL_NAME_STRUCTURE` for more.
 ```
 :::
 
+Names may have multiple parts that fit in the same piece; it is recommended that they appear in the same order in a piece structure as they do in the piece's superstructure.
+
+:::example
+```gedcom
+1 NAME William "Bill" Edward Hernandez y Martinez
+2 GIVN William Edward
+2 NICK Bill
+2 SURN Hernandez Martinez
```

Comment:
the above example requires explanation as to why it isn't "Hernandez y Martinez".  Otherwise it looks like a typo.

Reply:
It's not clear to me if this is the right example. Another example (a historical figure of some prominence) might be

```gedcom
1 NAME Pablo Diego José Francisco de Paula Juan Nepomuceno Crispín Cipriano de la Santísima Trinidad María de los Remedios Alarcón y Herrera Ruiz /Picasso/
2 GIVN Pablo Diego José Francisco de Paula Juan Nepomuceno Cipriano de la Santísima Trinidad
2 SURN Ruiz Picasso
```

but I'm not 100% sure I did that right. "Cipriano" and "María de los Remedios Alarcón y Herrera" were names added at baptism, not birth, but might still count as "given" names, depending on how you think of "given". Additionally, while it is clear he went by Picaso and that Ruiz was another surname, is slashes around one but both in SURN correct?